python/plugins/processing/algs/gdal/GdalUtils.py

```diff
@@ -19,13 +19,13 @@
 __date__ = "August 2012"
 __copyright__ = "(C) 2012, Victor Olaya"
 
-from typing import Dict, List, Optional
+from typing import Optional
 import os
-import subprocess
 import platform
 import re
-import warnings
 from dataclasses import dataclass
+import xml.etree.ElementTree as ET
```

Comment:
Can you use the Qt DOM classes for this instead of element tree? (It's one less library a QGIS maintainer is expected to be familiar with, and since we rely so heavily on the Qt classes it makes sense to stick with those wherever we can)